flow/project.py

```diff
@@ -1567,32 +1442,24 @@ def get(k, m):
         context['all_ops'] = all_ops
         context['parameters'] = parameters
         context['compact'] = compact
+        context['pretty'] = pretty
         context['unroll'] = unroll
         if overview:
             context['progress_sorted'] = progress_sorted
-            context['column_width_bar'] = column_width_bar
-            context['column_width_label'] = column_width_label
-            context['column_width_operation'] = column_width_operation
         if detailed:
-            context['column_width_id'] = column_width_id
-            context['column_width_operation'] = column_width_operation
-            context['column_width_total_label'] = column_width_total_label
             context['alias_bool'] = {True: 'T', False: 'U'}
```

Comment:
I know that this was introduced earlier, but I stumbled upon this while testing and don't think that the current aliases make much sense. What is 'T' and 'U' supposed to mean? I propose we replace this with 'Y' and 'N'.

Reply:
I agree. This was only used when the --expand option is used and it is used for eligible label of yes or no. So I think Y/N make better sense.